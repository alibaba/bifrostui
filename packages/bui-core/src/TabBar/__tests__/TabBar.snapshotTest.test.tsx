import { snapshotTest } from 'testing';

describe('TabBar snapshot', () => {
  snapshotTest('TabBar');
});
