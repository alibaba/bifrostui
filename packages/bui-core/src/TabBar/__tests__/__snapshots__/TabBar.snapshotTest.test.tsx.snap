// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TabBar snapshot TabBar demo snapshot 0 1`] = `
<div
  className="bui-stack"
  style={
    Object {
      "--align-items": "center",
      "--flex-direction": "column",
      "--flex-wrap": "nowrap",
      "--justify-content": "center",
      "--spacing": "0",
      "background": "#eee",
      "padding": "20px 0 10px",
    }
  }
>
  <div
    className="bui-tab-bar"
  >
    <div
      className="bui-tab-bar-item bui-tab-bar-item-active"
      onClick={[Function]}
    >
      <div
        className="bui-tab-bar-item-title bui-tab-bar-item-primary"
      >
        主页
      </div>
    </div>
    <div
      className="bui-tab-bar-item"
      onClick={[Function]}
    >
      <div
        className="bui-tab-bar-item-title bui-tab-bar-item-default"
      >
        商店
      </div>
    </div>
    <div
      className="bui-tab-bar-item"
      onClick={[Function]}
    >
      <div
        className="bui-tab-bar-item-title bui-tab-bar-item-default"
      >
        帮助
      </div>
    </div>
  </div>
</div>
`;

exports[`TabBar snapshot TabBar demo snapshot 0 2`] = `
<div
  className="bui-stack"
  style={
    Object {
      "--align-items": "center",
      "--flex-direction": "column",
      "--flex-wrap": "nowrap",
      "--justify-content": "center",
      "--spacing": "0",
      "background": "#eee",
      "padding": "20px 0 10px",
    }
  }
>
  <div
    className="bui-tab-bar"
  >
    <div
      className="bui-tab-bar-item bui-tab-bar-item-active"
      onClick={[Function]}
    >
      <svg
        className="bui-svg-icon"
        color="#ff335c"
        dangerouslySetInnerHTML={
          Object {
            "__html": "<path d=\\"m52.1 11.1 29.3 21.1c2.9 2.1 4.6 5.4 4.6 8.9V75c0 6.1-4.9 11-11 11H60V70c0-6.6-5.4-12-12-12s-12 5.4-12 12v16H21c-6.1 0-11-4.9-11-11V41.1c0-3.5 1.7-6.9 4.6-8.9l29.3-21.1c2.5-1.8 5.7-1.8 8.2 0z\\"/>",
          }
        }
        focusable="false"
        viewBox="0 0 96 96"
      />
      <div
        className="bui-tab-bar-item-title bui-tab-bar-item-primary"
      >
        主页
      </div>
    </div>
    <div
      className="bui-tab-bar-item"
      onClick={[Function]}
    >
      <svg
        className="bui-svg-icon"
        dangerouslySetInnerHTML={
          Object {
            "__html": "<path d=\\"M68.7 80H27.3c-1.8 0-3.3-1.8-3.3-4s1.5-4 3.3-4h41.4c1.8 0 3.3 1.8 3.3 4s-1.5 4-3.3 4m15.4-26.7c-1 .1-1.9.2-2.9.2-5.2 0-10.3-1.7-14.4-4.7-1.3-1-3.1-1-4.4 0-4.1 3.1-9.2 4.8-14.4 4.7-5.2 0-10.3-1.7-14.4-4.7-1.3-1-3.1-1-4.4 0-4.9 3.6-11.2 5.2-17.3 4.5-1-.1-2 .2-2.7.8S8 55.7 8 56.6v28.6C8 89 11.1 92 14.9 92h66.3c3.8 0 6.9-3 6.9-6.8V56.6c-.1-2-1.9-3.5-4-3.3m10.6-28L87.1 9.9C85.2 6.3 81.6 4 77.7 4H18.3c-3.9 0-7.6 2.3-9.4 5.9L1.3 25.3c-4.2 10 2 21.8 12.6 22.7 5.9.5 11.6-2.3 14.9-7.3.6-1 1.7-1.5 2.8-1.5 1.1 0 2.2.6 2.8 1.5C37.5 45.2 42.6 48 48 48s10.5-2.8 13.5-7.4c.6-1 1.7-1.6 2.8-1.6s2.2.6 2.8 1.6c3.3 5 9 7.8 14.9 7.3 10.6-.8 16.9-12.6 12.7-22.6\\"/>",
          }
        }
        focusable="false"
        viewBox="0 0 96 96"
      />
      <div
        className="bui-tab-bar-item-title bui-tab-bar-item-default"
      >
        商店
      </div>
    </div>
    <div
      className="bui-tab-bar-item"
      onClick={[Function]}
    >
      <svg
        className="bui-svg-icon"
        dangerouslySetInnerHTML={
          Object {
            "__html": "<path d=\\"M68.3 56.5c9.1 3.3 13.8 13.2 10.8 22.3l-.3.8C75.6 88.5 64.8 93 56.4 90l-4-1.5h-.1c-12.8-5.9-24.4-16.6-30-27.8-6.7-13.2-8.4-30.3-3.5-43.9C22.4 7.1 32.2 2.7 41.7 6.2c9.3 3.4 14 13.6 10.6 22.9-2.4 6.5-8.2 10.5-14.9 11 .8 3.8 2.1 7.6 3.9 11.2C43.4 55.4 46 59 49 62c4.7-5.7 12.1-8.1 19.3-5.5z\\"/>",
          }
        }
        focusable="false"
        viewBox="0 0 96 96"
      />
      <div
        className="bui-tab-bar-item-title bui-tab-bar-item-default"
      >
        帮助
      </div>
    </div>
  </div>
</div>
`;

exports[`TabBar snapshot TabBar demo snapshot 0 3`] = `
Array [
  <div
    className="bui-stack"
    style={
      Object {
        "--align-items": "flex-start",
        "--flex-direction": "column",
        "--flex-wrap": "nowrap",
        "--justify-content": "center",
        "--spacing": "0",
        "background": "#eee",
        "padding": "20px 0 10px",
      }
    }
  >
    <div
      style={
        Object {
          "margin": "8px",
        }
      }
    >
      自定义选中主题色: 
    </div>
    <div
      className="bui-tab-bar"
    >
      <div
        className="bui-tab-bar-item bui-tab-bar-item-active"
        onClick={[Function]}
      >
        <div
          className="bui-tab-bar-item-title bui-tab-bar-item-primary"
        >
          主页
        </div>
      </div>
      <div
        className="bui-tab-bar-item"
        onClick={[Function]}
      >
        <div
          className="bui-tab-bar-item-title bui-tab-bar-item-default"
        >
          商店
        </div>
      </div>
      <div
        className="bui-tab-bar-item"
        onClick={[Function]}
      >
        <div
          className="bui-tab-bar-item-title bui-tab-bar-item-default"
        >
          帮助
        </div>
      </div>
    </div>
  </div>,
  <div
    className="bui-stack"
    style={
      Object {
        "--align-items": "flex-start",
        "--flex-direction": "column",
        "--flex-wrap": "nowrap",
        "--justify-content": "center",
        "--spacing": "0",
        "background": "#eee",
        "padding": "20px 0 10px",
      }
    }
  >
    <div
      style={
        Object {
          "margin": "8px",
        }
      }
    >
      自定义默认主题色: 
    </div>
    <div
      className="bui-tab-bar"
    >
      <div
        className="bui-tab-bar-item bui-tab-bar-item-active"
        onClick={[Function]}
      >
        <div
          className="bui-tab-bar-item-title bui-tab-bar-item-primary"
        >
          主页
        </div>
      </div>
      <div
        className="bui-tab-bar-item"
        onClick={[Function]}
      >
        <div
          className="bui-tab-bar-item-title bui-tab-bar-item-info"
        >
          商店
        </div>
      </div>
      <div
        className="bui-tab-bar-item"
        onClick={[Function]}
      >
        <div
          className="bui-tab-bar-item-title bui-tab-bar-item-info"
        >
          帮助
        </div>
      </div>
    </div>
  </div>,
]
`;

exports[`TabBar snapshot TabBar demo snapshot 0 4`] = `
<div
  className="bui-stack"
  style={
    Object {
      "--align-items": "center",
      "--flex-direction": "column",
      "--flex-wrap": "nowrap",
      "--justify-content": "center",
      "--spacing": "0",
      "background": "#eee",
      "padding": "20px 0 10px",
    }
  }
>
  <div
    className="bui-tab-bar"
  >
    <div
      className="bui-tab-bar-item bui-tab-bar-item-active"
      onClick={[Function]}
    >
      <svg
        className="bui-svg-icon"
        color="#ff335c"
        dangerouslySetInnerHTML={
          Object {
            "__html": "<path d=\\"m52.1 11.1 29.3 21.1c2.9 2.1 4.6 5.4 4.6 8.9V75c0 6.1-4.9 11-11 11H60V70c0-6.6-5.4-12-12-12s-12 5.4-12 12v16H21c-6.1 0-11-4.9-11-11V41.1c0-3.5 1.7-6.9 4.6-8.9l29.3-21.1c2.5-1.8 5.7-1.8 8.2 0z\\"/>",
          }
        }
        focusable="false"
        viewBox="0 0 96 96"
      />
      <div
        className="bui-tab-bar-item-title bui-tab-bar-item-primary"
      >
        主页
      </div>
    </div>
    <div
      className="bui-tab-bar-item"
      onClick={[Function]}
    >
      <div
        className="bui-badge bui-badge-position"
      >
        <div
          className="bui-badge-primary bui-badge-contained bui-badge-dot"
        />
        <svg
          className="bui-svg-icon"
          dangerouslySetInnerHTML={
            Object {
              "__html": "<path d=\\"M68.7 80H27.3c-1.8 0-3.3-1.8-3.3-4s1.5-4 3.3-4h41.4c1.8 0 3.3 1.8 3.3 4s-1.5 4-3.3 4m15.4-26.7c-1 .1-1.9.2-2.9.2-5.2 0-10.3-1.7-14.4-4.7-1.3-1-3.1-1-4.4 0-4.1 3.1-9.2 4.8-14.4 4.7-5.2 0-10.3-1.7-14.4-4.7-1.3-1-3.1-1-4.4 0-4.9 3.6-11.2 5.2-17.3 4.5-1-.1-2 .2-2.7.8S8 55.7 8 56.6v28.6C8 89 11.1 92 14.9 92h66.3c3.8 0 6.9-3 6.9-6.8V56.6c-.1-2-1.9-3.5-4-3.3m10.6-28L87.1 9.9C85.2 6.3 81.6 4 77.7 4H18.3c-3.9 0-7.6 2.3-9.4 5.9L1.3 25.3c-4.2 10 2 21.8 12.6 22.7 5.9.5 11.6-2.3 14.9-7.3.6-1 1.7-1.5 2.8-1.5 1.1 0 2.2.6 2.8 1.5C37.5 45.2 42.6 48 48 48s10.5-2.8 13.5-7.4c.6-1 1.7-1.6 2.8-1.6s2.2.6 2.8 1.6c3.3 5 9 7.8 14.9 7.3 10.6-.8 16.9-12.6 12.7-22.6\\"/>",
            }
          }
          focusable="false"
          viewBox="0 0 96 96"
        />
        <div
          className="bui-tab-bar-item-title bui-tab-bar-item-default"
        >
          商店
        </div>
      </div>
    </div>
    <div
      className="bui-tab-bar-item"
      onClick={[Function]}
    >
      <div
        className="bui-badge bui-badge-position"
      >
        <div
          className="bui-badge-primary bui-badge-contained bui-badge-standard"
        >
          20
        </div>
        <svg
          className="bui-svg-icon"
          dangerouslySetInnerHTML={
            Object {
              "__html": "<path d=\\"M68.3 56.5c9.1 3.3 13.8 13.2 10.8 22.3l-.3.8C75.6 88.5 64.8 93 56.4 90l-4-1.5h-.1c-12.8-5.9-24.4-16.6-30-27.8-6.7-13.2-8.4-30.3-3.5-43.9C22.4 7.1 32.2 2.7 41.7 6.2c9.3 3.4 14 13.6 10.6 22.9-2.4 6.5-8.2 10.5-14.9 11 .8 3.8 2.1 7.6 3.9 11.2C43.4 55.4 46 59 49 62c4.7-5.7 12.1-8.1 19.3-5.5z\\"/>",
            }
          }
          focusable="false"
          viewBox="0 0 96 96"
        />
        <div
          className="bui-tab-bar-item-title bui-tab-bar-item-default"
        >
          帮助
        </div>
      </div>
    </div>
  </div>
</div>
`;
